x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -365,7 +365,7 @@ func (k Keeper) chargeComputeUnitsToProjectAndSubscription(ctx sdk.Context, clie
 		return fmt.Errorf("failed to add CU to the project")
 	}
 
-	err = k.subscriptionKeeper.ChargeComputeUnitsToSubscription(ctx, project.GetSubscription(), relay.CuSum)
+	err = k.subscriptionKeeper.ChargeComputeUnitsToSubscription(ctx, project.GetSubscription(), uint64(ctx.BlockHeight()), relay.CuSum)
```

Comment:
should be the block of the relay, relay.block

Reply:
a007fabe